ProjFS.Mac/PrjFSLib/PrjFSLib.cpp

```diff
@@ -1057,6 +1057,7 @@ static PrjFS_Result HandleFileNotification(
         request->pid,
         request->procname,
         isDirectory,
+        placeholderFile,
         notificationType,
         nullptr /* destinationRelativePath */);
     
```

Comment:
The code below (for PrjFS_NotificationType_PreConvertToFull) should also happen for file modified notifications when `placeholderFile` is true.